.github/workflows/test.yml

```diff
@@ -392,9 +392,11 @@ jobs:
               issue_number: context.issue.number,
               owner: context.repo.owner,
               repo: context.repo.repo,
-              body: `Hello! We're in the process of migrating to [nf-test](https://code.askimed.com/nf-test)!🚀
+              body: 'Hello! We're in the process of migrating to [nf-test](https://code.askimed.com/nf-test)!🚀
```

Comment:
Will it fail if we have single quotes in the text? I think we should use `

Reply:
Using double quotes now